File: ClosureDefFibonacci.js
function ClosureDefFibonacci() {
}
ClosureDefFibonacci.prototype.bench = function() {
  this.fib(25);
}
ClosureDefFibonacci.prototype.fib = function(x) {
  var self = this;
  function f(n) {
    return n < 2 ? 1 : self.fib(n-1) + self.fib(n-2);
  }
  return f(x);
}
global.ClosureDefFibonacci = ClosureDefFibonacci;

File: ClosureDefFibonacci.dart
class ClosureDefFibonacci {
  bench() {
    this.fib(25);
  }
  fib(x) {
    f(n) => n < 2 ? 1 : fib(n-1) + fib(n-2);
    return f(x);
  }
}

